app/models/version_footer.rb

```diff
@@ -40,7 +40,6 @@ def self.stale?
     else
       @@stale = true
     end
-    @@stale
```

Comment:
I believe that this is redundant with the above `if/else` block

Reply:
👍🏻